### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by jgrexa** [timescale/timescale-prometheus#28]
I encountered following issue during testing the adapter. It was working for several hours, then the pods got into crashloop:
```
{"caller":"log.go:43","level":"info","msg":"Samples write throughput","samples/sec":67,"ts":"2020-04-15T12:57:02.440Z"}
{"caller":"log.go:43","level":"info","msg":"Samples write throughput","samples/sec":1,"ts":"2020-04-15T12:57:03.273Z"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xaf002e]

goroutine 53 [running]:
github.com/timescale/timescale-prometheus/pkg/pgmodel.(*orderedMap).Front(...)
	/go/timescale-prometheus-connector/pkg/pgmodel/pgx.go:654
github.com/timescale/timescale-prometheus/pkg/pgmodel.(*insertHandler).flushTimedOutReqs(0xc0001c8000)
	/go/timescale-prometheus-connector/pkg/pgmodel/pgx.go:549 +0x4e
github.com/timescale/timescale-prometheus/pkg/pgmodel.runInserterRoutine(0xe3e1a0, 0xc0000ba028, 0xc0000bc4e0)
	/go/timescale-prometheus-connector/pkg/pgmodel/pgx.go:504 +0x322
created by github.com/timescale/timescale-prometheus/pkg/pgmodel.newPgxInserter
	/go/timescale-prometheus-connector/pkg/pgmodel/pgx.go:237 +0xaa
```

**Comment by cevian** [comment]
Thank you @jgrexa for the report. I believe this is fixed in #24 . Can @JLockerman confirm?

**Comment by jgrexa** [comment]
I have compiled latest version and the problem no longer appears.
